README.md

```diff
@@ -239,6 +239,13 @@ To install libCEED for Python, run
 
     python setup.py build install
 
+with the desired setuptools options, such as `--user`.
+
+Alternatively, if libCEED in installed in the directory specified by environment
```

Comment:
Is installed...
And
... specified by the environment variable

Reply:
Thanks